### DISCUSSION TITLE
Error when determining the Sel number

### DISCUSSION THREAD

**Discussion by hu-lm** [deepmodeling/deepmd-kit#1816] [Q&A] [answered]
Hi,

I was trying to use the command from section 'determine Sel' https://github.com/deepmodeling/deepmd-kit/blob/master/doc/model/sel.md  
However, deepmd returns an error message like this
![image](https://user-images.githubusercontent.com/98113070/177426262-d6898e9c-674a-4686-a4a8-ce40a2042ade.png)

Indicating that the command dp neighbor-stat is no longer a valid argument choice. Thus I would like to know if there is still a command to determine the proper Sel? If not, which equation should I refer to in order to compute the Sel for my system? Thank you so much!

Best,
LM

**Comment by njzjz** [comment, marked as answer]
You may use a newer version.
